actors/builtin/miner/monies.go

```diff
@@ -3,15 +3,59 @@ package miner
 import (
 	"github.com/filecoin-project/specs-actors/actors/abi"
 	"github.com/filecoin-project/specs-actors/actors/abi/big"
+	"github.com/filecoin-project/specs-actors/actors/builtin"
 )
 
+// IP = InitialPledgeFactor * BR(precommit time)
+var InitialPledgeFactor = big.NewInt(20)
+
+// FF = (DeclaredFaultFactorNum / DeclaredFaultFactorDenom) * BR(t)
+var DeclaredFaultFactorNum = big.NewInt(214)
+
+// FF = (DeclaredFaultFactorNum / DeclaredFaultFactorDenom) * BR(t)
+var DeclaredFaultFactorDenom = big.NewInt(100)
+
+// SP = UndeclaredFaultFactor * BR(t)
+var UndeclaredFaultFactor = big.NewInt(5)
+
+// This is the BR(t) value of the given sector for the current epoch.
+// It is the expected reward this sector would pay out over a one day period.
+// BR(t) = CurrEpochReward(t) * SectorQualityAdjustedPower * EpochsInDay / TotalNetworkQualityAdjustedPower(t)
+func ExpectedDayRewardForSector(epochTargetReward abi.TokenAmount, networkQAPower abi.StoragePower, qaSectorPower abi.StoragePower) abi.TokenAmount {
```

Comment:
This function isn't about sectors, it's just about power. Rename to something like `ExpectedDayRewardForPower`.